plugins/kubernetes/src/components/Pods/PodsTable.tsx

```diff
@@ -67,7 +68,11 @@ const READY: TableColumn<V1Pod>[] = [
   },
 ];
 
-export const PodsTable = ({ pods, extraColumns = [] }: PodsTablesProps) => {
+export const PodsTable = ({
```

Comment:
There is an established pattern for column customization that I think it would be best to adhere to here. It's to have a [`columns`](https://github.com/backstage/backstage/blob/1a62c3fec9e33a677bf90ed4f84d6510a1fa5f44/plugins/catalog/src/components/CatalogTable/CatalogTable.tsx#L42) property that replaces all of the existing columns, and then provide [factories](https://github.com/backstage/backstage/blob/master/plugins/catalog/src/components/CatalogTable/columns.tsx) for all of the existing columns.

Reply:
+1 on the above - and I'd have expected a columns field to be added at the bottom, [like this one](https://github.com/backstage/backstage/blob/805751631abdf41142373c2f738e0e4d95495653/plugins/catalog/src/components/CatalogTable/CatalogTable.tsx#L170).